### PR TITLE
Feature/55 language counter

### PR DESCRIPTION
**Pull request by kmcelwee**
Add column in language admin view to show the document count of the given language. Points the link to a filtered view of (borrowed from s&co!) Addresses #55, paired with @thatbudakguy

### CODE REVIEW

File: geniza/corpus/tests/test_corpus_admin.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+from django.test import TestCase
+from django.contrib import admin
+
+from geniza.corpus.models import LanguageScript, Document
+from geniza.corpus.admin import LanguageScriptAdmin
+
+class TestLanguageScriptAdmin(TestCase):
+    def test_borrows(self):
```
Comment (rlskoeser) [inline geniza/corpus/tests/test_corpus_admin.py:8]: test borrows?

File: geniza/corpus/admin.py [hunk 2]
```diff
@@ -12,8 +15,15 @@ class CollectionAdmin(admin.ModelAdmin):
 
 @admin.register(LanguageScript)
 class LanguageScriptAdmin(admin.ModelAdmin):
-    list_display = ('language', 'script', 'display_name')
-
+    list_display = ('language', 'script', 'display_name', 'usage_count')
+
+    def usage_count(self, obj):
```
Comment (rlskoeser) [inline geniza/corpus/admin.py:20]: This label seems a little odd to me. What about documents? Display label could be '# documents'

**Review by rlskoeser — APPROVED** [review]
Changes look good, just a couple comments about names

File: geniza/corpus/tests/test_corpus_admin.py (hunk 1 above)
Comment (kmcelwee) [reply geniza/corpus/tests/test_corpus_admin.py]: whoops, copied and pasted from s+co. Good catch! I've addressed everything else. @rebeccakoeser please confirm that I should merge

**Pull request merged**